packages/flutter/lib/src/cupertino/theme.dart

```diff
@@ -144,6 +172,24 @@ class _InheritedCupertinoTheme extends InheritedWidget {
   bool updateShouldNotify(_InheritedCupertinoTheme old) => theme.data != old.theme.data;
 }
 
+/// Convert the [extensionsIterable] passed to [CupertinoThemeData.new] or [copyWith]
+/// to the stored [extensions] map, where each entry's key consists of the extension's type.
+Map<Object, CupertinoThemeExtension<dynamic>> _themeExtensionIterableToMap(Iterable<CupertinoThemeExtension<dynamic>> extensionsIterable) {
+  return Map<Object, CupertinoThemeExtension<dynamic>>.unmodifiable(<Object, CupertinoThemeExtension<dynamic>>{
+    // Strangely, the cast is necessary for tests to run.
+    for (final CupertinoThemeExtension<dynamic> extension in extensionsIterable) extension.type: extension as CupertinoThemeExtension<CupertinoThemeExtension<dynamic>>,
+  });
+}
+
+/// Returns a copy of the current [extensions] with all the fields resolved 
```

Comment:
Looks like the analyze test is complaining about a trailing space at the end of this line

Reply:
fixed.